src/nxdoc/nuxeo-server/nuxeo-server-release-notes/nuxeo-server-release-notes-2021-11.md

```diff
@@ -0,0 +1,54 @@
+---
+title: LTS 2021.11 / LTS 2021-HF11
+description: Discover what's new in LTS 2021.11 / LTS 2021-HF11
+review:
+   comment: ''
+   date: '2021-11-08'
+   status: ok
+labels:
+    - release-notes
+toc: true
+tree_item_index: 5300
+hidden: true
+---
+
+{{! multiexcerpt name='nuxeo-server-updates-2021-11'}}
+# What's New in LTS 2021.11 / LTS 2021-HF11
+
+## Nuxeo Server
+
+### Core Storage
+
+#### `SystemDocument` on Core Management Documents
+
+The facet `SystemDocument` has been added on core management documents.
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-30544](https://jira.nuxeo.com/browse/NXP-30544)
+
+#### Bulk Action for Mongo Read ACL Propagation
+
+Mongo Read ACL propagation are now updated with a BAF action.
+
```

Comment:
Suggestion: There are several benefits to this change: improve the performances for larges repositories, preserve the platform availability during the ACL propagation, improve the monitoring and the error management,...